studio/qclient_cache.py

```diff
@@ -1,27 +1,35 @@
 import threading
 
-from datetime import timedelta
-
-from cachetools import LRUCache
 from .rabbit_queue import RMQueue
 
-queue_cache = {}
-
+_queue_cache = {}
 
 def get_cached_queue(
         name,
         route,
         cloud=None,
         config=None,
         logger=None,
-        close_after=timedelta(),
+        close_after=None,
         verbose=10):
+
```

Comment:
Implement very basic queues cache - just a dictionary {name => RMQueue object).
This way we are no longer creating a new RMQueue object for every payload
we send through completion service.